napari/components/experimental/chunk/__init__.py

```diff
@@ -1,22 +1,22 @@
 """chunk module"""
-from napari.components.experimental.chunk._loader import (
-    chunk_loader,
-    synchronous_loading,
-    wait_for_async,
-)
-from napari.components.experimental.chunk._request import (
-    ChunkLocation,
-    ChunkRequest,
-    LayerRef,
-    OctreeLocation,
-)
+# from napari.components.experimental.chunk._loader import (
```

Comment:
Is there a reason to change this package initializer? My instinct is just to leave it alone.

Reply:
I removed all of the config. Because everything is intertwined, some classes were getting initialized on some imports. It was very hairy. My thinking was that if we comment this file, we can keep it all intact without sprinkling changes throughout all the octree methods. Re-enabling then becomes uncommenting these __init__.py and adding back the config in a few places.